addons/interaction/XEH_preStart.sqf

```diff
@@ -1,3 +1,25 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+if (!hasInterface) exitWith {};
+
+private _replaceTerrainClasses = QUOTE( \
+    getNumber (_x >> 'scope') == 2 \
+    && {getNumber (_x >> QQGVAR(replaceTerrainObject)) > 0} \
+) configClasses (configFile >> "CfgVehicles");
```

Comment:
```suggestion
private _replaceTerrainClasses = QUOTE(
    getNumber (_x >> 'scope') == 2
    && {getNumber (_x >> QQGVAR(replaceTerrainObject)) > 0}
) configClasses (configFile >> "CfgVehicles");
```
This should work just fine as it is not a macro definition.

Reply:
you are right, it works. I also swapped conditions and got -10ms :)